core.go

```diff
@@ -68,10 +100,30 @@ func sendHttpStatusOk(writer http.ResponseWriter) {
 	writer.WriteHeader(http.StatusOK)
 }
 
+func helpHttpHandler(writer http.ResponseWriter, response *http.Request) {
+	sendHttpStatusOk(writer)
+	sendHelpMessageToClient(writer)
+}
+
 func sendByeMessageToClient(writer http.ResponseWriter) {
 	writer.Write([]byte("bye"))
 }
 
+func readHelpFile() string {
+	pwd, _ := os.Getwd()
+	dat, err := ioutil.ReadFile(pwd+"/../src/core/help.txt")
```

Comment:
thoughts-

1. Maybe compile this into the binary
2. Use an environment variable
3.